dev/main.ts

```diff
@@ -15,6 +22,54 @@ import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap-vue/dist/bootstrap-vue.css';
 import 'bootstrap-vue/dist/bootstrap-vue-icons.min.css';
 
+class Invoice {
+  @DataGridField('String')
+  id!: string;
+
+  @DataGridField('String')
+  firstName!: string;
+
+  @DataGridField('String')
+  lastName!: string;
+
+  @DataGridField('String')
+  email!: string;
+
+  @DataGridField('Lang')
+  language!: string;
+
+  @DataGridField('Number')
+  note!: number;
+
+  @DataGridField('Money')
+  total!: number;
+
+  @DataGridField('InvoiceStatus')
+  status!: string;
+
+  @DataGridField('Date')
+  createdAt!: Date;
+}
+class Student {
+  @DataGridField('String')
+  id!: string;
+
+  @DataGridField('String', { name: 'Prenom' })
```

Comment:
`label` is probably better than `name` if only used for text display

Reply:
Je viens de me rendre compte que c'est liée au QueryBuilder qu'on utilise comme dependance. Je vais changer sur le décorateur. Mais dans la pratique le créateur de règles va  créer ce champs.

Reply:
On va mettre a la poubelle le UI du QueryDesigner. Ça semblait bien, mais finalement c'est vraiment pas intuitif.

Perso, si on peut refaire quelque chose de similaire au BAP ça serait top. ( avec le drag and drop.

Faut pouvoir ajouter 15-20 conditions emet que ça soit utilisable.